Demo/Application/Features/PayPalNativeCheckoutViewController.swift

```diff
@@ -2,6 +2,7 @@ import Foundation
 import UIKit
 import BraintreePayPalNativeCheckout
 
+@available(*, deprecated, message: "BraintreePayPalNativeCheckout Module is deprecated, use BraintreePayPal Module instead")
```

Comment:
This is our demo app, we don't need a deprecation message here, just in the public classes

Reply:
yes, but I got lint errors from referencing MXO classes here

Reply:
We don't have a linter on Braintree. Do you mean Cocoapods `pod lib lint`?

Reply:
yes. I think so.

Reply:
https://github.com/braintree/braintree_ios/actions/runs/9614991255/job/26521134948

Reply:
Looks ok without the line there! Are we ok with the lint warning here?

Reply:
addressed here: 991982b41a3dd722ba7c0c40ae1aab7861a0985a